polishNotationCalc.cpp

```diff
@@ -0,0 +1,92 @@
+#include "polishNotationCalc.hpp"
+namespace dev
+{
+    PolishCalc::PolishCalc()
+    {
+        insertFunctionInsideMap();
+    }
+
```

Comment:
**1 Step : Create Token  stack and validate it** 

token_stack& =  stack<Tocken>

Token {
  // Process one tocken in the stack 
    void process(token_stack& ) 
    double value(const token_stack& )  << // Follow contant 
}

NumericToken  : Token {
  void  process(token_stack& ) -> do nothing
  void  value(token_stack& ) -> return value
  double value_
} 

OperatorTocken : Token {
  pair PopTockens 
}

PlusTocken  : OperatorTocken {
  void  process(token_stack& ) { 
     pop token from stack -> get value -> double res1
     pop token2 from stack -> get value -> double res2
     res1 + res2 -> Plus result 
     Push plus result to token vector  as Numeric 
  }
  
  double  value(const token_stack&) {
    // Option 1 Contract thet process() should be evaluated before value ()
     return top of the token_stack& value() 
  
    // Option 2 Base that there process() could not be called
         get token from stack -> get value -> double res1
         get token2 from stack on the second position -> get value -> double res2 
         res1 + res2 -> Plus result 
         return Plus result  as double 
  } 
} 

Go throw